src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+
+CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';CREATE DATABASE IF NOT EXISTS airlines;
+CREATE USER 'user'@'localhost' IDENTIFIED BY 'user';
+GRANT ALL PRIVILEGES ON airlines.* TO 'user'@'localhost';
+
+USE airlines;
+
+DROP TABLE IF EXISTS `airplanes`;
+DROP TABLE IF EXISTS `crews_crew_members`;
+DROP TABLE IF EXISTS `crews`;
+DROP TABLE IF EXISTS `crew_members`;
+
+CREATE TABLE `crews` (
+                         `id` INT AUTO_INCREMENT PRIMARY KEY,
+                         `name` VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE `airplanes` (
+                             `id` INT AUTO_INCREMENT PRIMARY KEY,
+                             `code name` VARCHAR(10) NOT NULL,
+                             `model` VARCHAR(30) NOT NULL,
+                             `manufacture date` DATE NOT NULL,
+                             `capacity` INT NOT NULL,
+                             `flight range` INT NOT NULL,
+                             `crews_id` INT,
+                             CONSTRAINT `FK_airplanes_crews` FOREIGN KEY (`crews_id`) REFERENCES crews(`id`)
+);
+
+CREATE TABLE `crew_members` (
+                                `id` INT AUTO_INCREMENT PRIMARY KEY,
+                                `first name` VARCHAR(75) NOT NULL,
```

Comment:
In Intellij IDEA in this file you could choose SQL Dialect to help Idea with Syntax warning and auto-filling and formatting.

You could do left-click by mouse and pick menu item "Change dialect (SQL) ..." and then choose MySQL.

Then press CTRL+A (select all) and CTRL+ALT+L for autoformatting

Reply:
fixed in